cassandra/pom.xml

```diff
@@ -88,8 +88,8 @@
         </dependency>
 
         <dependency>
-            <groupId>net.jpountz.lz4</groupId>
-            <artifactId>lz4</artifactId>
+            <groupId>org.lz4</groupId>
+            <artifactId>lz4-java</artifactId>
```

Comment:
we need to update https://github.com/apache/zeppelin/blob/79108419e956d4774f87303d970888a8c46132f8/zeppelin-distribution/src/bin_license/LICENSE#L71

Reply:
done...
Thank you for review